CONTRIBUTING.md

```diff
@@ -362,11 +362,13 @@ You can also remove your `$HOME/nix-installer-wsl-tests-temp` folder whenever yo
 To cut a release:
 
 * Ensure the `flake.lock`, `Cargo.lock`, and Rust dependencies are up-to-date with the following:
-  + `nix flake update --commit-lock-file`
-  + `cargo update` (and commit)
+  + `nix flake update`
```

Comment:
I removed this arg because it felt unnecessary, we squash merge anyways. (similarly, the commit below)